PanelViewController/Framework source/PanelViewController.swift

```diff
@@ -292,12 +419,19 @@ extension PanelViewController: DraggableViewDelegate {
     func draggingBegan(view: DraggableView) {
         animator.removeAllBehaviors()
         isDragging = true
-        
-        slidingViewController?.view.frame = CGRect(x: 0, y: closedHeight, width: paneView.bounds.width, height: view.bounds.height - closedHeight)
+        slidingViewController?.view.frame = CGRect(x: 0, y: closedHeight + floatingHeaderHeight, width: paneView.bounds.width, height: view.bounds.height - closedHeight)
     }
     
     func draggingEnded(view: DraggableView, velocity: CGPoint) {
         isDragging = false
         performStateChange(velocity: velocity)
     }
+    
+    func shouldDrag(view: DraggableView, location: CGPoint) -> Bool {
```

Comment:
What is this used for?  Maybe I comment?

Reply:
I added a comment as you recommended